cogs/highlights.py

```diff
@@ -37,7 +37,23 @@ def _should_send_highlight(msg: discord.Message, member: discord.Member, query:
     )
 
 
+class OptionalGuildConverter(commands.converter.GuildConverter):
+    async def convert(self, ctx: commands.Context, argument: str) -> discord.Guild | None:
+        if argument == "0":
+            return None
+        return await super().convert(ctx, argument)
+
+
+OptionalCurrentGuild = commands.parameter(
+    default=lambda ctx: ctx.guild,
+    displayed_default="<this server>",
+    converter=OptionalGuildConverter,
+)
```

Comment:
These should be moved to /core/converters.py. The converter class needs a docstring

Reply:
Done.